site/smart-accounts/light-account/index.md

```diff
@@ -33,18 +35,46 @@ The code snippet below demonstrates how to use Light Account with Account Kit. I
 
 ::: code-group
 
+## Choosing a version
+
+Light Account comes in versions v1.1.0 and v2.0.0, which make use of the v0.6 and v0.7 entry points respectively. For more information about the differences between entry points, see the [EntryPoint v0.7 Upgrade Guide](/using-smart-accounts/entry-point-v7.html).
+
+For backwards compatibility, Light Account defaults to version v1.1.0, but we recommend v2.0.0 for new dapps because of the advantages that come with entry point v0.7. However, once a version is chosen and the light account is created, the version must remain consistent in order for the light account client to work with the existing light account.
+
 ::: tip Address calculation
-For the Light Account, the address of the smart account will be calculated as a combination of [the owner and the salt](https://github.com/alchemyplatform/light-account/blob/main/src/LightAccountFactory.sol#L24-L33). You will get the same smart account address each time you supply the same `owner`, the signer used to create the account for the first time. Alternatively, you can supply `salt` if you want a different address for the same `owner` param (the default salt is `0n`).
+For the Light Account, the address of the smart account will be calculated as a combination of the version, [the owner, and the salt](https://github.com/alchemyplatform/light-account/blob/main/src/LightAccountFactory.sol#L24-L33). You will get the same smart account address each time you supply the same `version` and `owner`. Alternatively, you can supply `salt` if you want a different address for the same `version` and `owner` params (the default salt is `0n`).
```

Comment:
Nit: pin to v2.0.0 tag https://github.com/alchemyplatform/light-account/blob/v2.0.0/src/LightAccountFactory.sol#L24-L33